content/en/docs/signing_and_verification.md

```diff
@@ -0,0 +1,214 @@
+
+4. **Signature Verification**:
+- When an image is pulled from Harbor, the system verifies its signature using the corresponding public key.
+- If the signature is invalid or absent, the image is flagged or rejected, preventing unauthorized or tampered images from being used.
+
+---
+
+## Benefits of Using Notation and Harbor Together
+
+By combining the signing capabilities of Notation with the secure registry features of Harbor:
+- Organizations can effectively prevent tampering.
+- Only trusted images are used in deployments.
+- Software supply chains are strengthened, particularly in environments where security is critical (e.g., Kubernetes deployments).
+
+This approach mitigates risks posed by unsigned or unverified images, ensuring secure and trusted deployments.
+
+![Workflow Representation of Signing and Verifying Container Images](https://www.redhat.com/rhdc/managed-files/styles/wysiwyg_full_width/private/ohc/Signing%20and%20verifying%20container%20images-2.jpeg.webp?itok=Yd-qjm25 "Workflow Representation of Signing and Verifying Container Images")
+
+# Signing and Verifying Docker Images with Notation and Harbor 
+
+## Why is it Important? 
+
+### 1. Ensures Integrity 
+Signing Docker images guarantees that the image contents have not been altered or tampered with after being signed. Verification ensures the image is in its original state as intended by the creator. 
+
+### 2. Authenticates Source 
+By signing images, producers provide cryptographic proof of their identity. Consumers can verify the signature to ensure the image comes from a trusted source, reducing the risk of using malicious or counterfeit software. 
+
+### 3. Prevents Tampering 
+The cryptographic signature attached to the image serves as a safeguard against unauthorized modifications. If even a single byte of the image is changed, the verification process will fail, signaling potential tampering. 
+
+---
+
+## Pre-requisites for Setting Up Harbor and Notation 
+
+Before diving into the setup and configuration of Harbor and Notation, it is crucial to ensure that some foundational tools and infrastructure are in place. 
+
+### 1. Docker Engine 
+- **Description**: The core component for creating and running containers. It allows for the building, shipping, and running of containerized applications. 
+- **Importance**: Docker Engine is required to interact with Harbor and manage the lifecycle of container images. 
+
+### 2. Docker Compose 
+- **Description**: A tool that simplifies the process of setting up and managing multi-container applications. 
+- **Usage**: In the context of Harbor, Docker Compose will be used to orchestrate the deployment of various Harbor components, ensuring that all services are configured and running correctly. 
+
+> **Note**: Before proceeding with Harbor installation, both Docker Engine and Docker Compose must be properly installed and configured on the host system. 
+
+---
+
+## Setting Up Harbor with HTTPS Domain Configuration 
```

Comment:
I would suggest add a link from Harbor on setting up harbor, as this document should focus on signing with Notation, signatures are stored in Harbor referrering to the container images, verifying images, and What is special on Harbor policies.